driver-core/src/main/com/mongodb/client/model/search/SearchOperator.java

```diff
@@ -41,6 +57,24 @@ static ExistsSearchOperator exists(final FieldSearchPath path) {
         return new SearchConstructibleBsonElement("exists", new BsonDocument("path", (notNull("path", path)).toBsonValue()));
     }
 
+    /**
+     * Returns a {@link SearchOperator} that performs a full-text search.
+     *
+     * @param queries Non-empty terms to search for.
+     * @param paths Non-empty document fields to be searched.
+     * @return The requested {@link SearchOperator}.
+     * @mongodb.atlas.manual atlas-search/text/ text operator
+     */
+    static TextSearchOperator text(final Iterable<String> queries, final Iterable<? extends SearchPath> paths) {
```

Comment:
I know you're not fond of redundant overloads, but this signature could get annoying if the common case is to have a single query and path.

Reply:
Done.